source/ways-of-working/standards/flyway-database-migrations.html.md.erb

```diff
@@ -0,0 +1,126 @@
+---
+title: Flyway database migration
+last_reviewed_on: 2022-03-17
+review_in: 6 months
+weight: 10
+---
+
+# <%= current_page.data.title %>
+
+## How to apply Flyway migration as part of pipeline
+
+Enable database migration for Jenkins:
+
+- Add `enableDbMigration('<PROJECT_NAME>')` to Jenkinsfile_CNP file
+
+Add correct config to `build.gradle`:
+
+- Ensure the latest flyway version is set e.g. `id 'org.flywaydb.flyway' version '<VERSION_NUMBER>'`
+- Add this block:
+
+```
+flyway {
+  url = System.getenv('FLYWAY_URL')
+  user = System.getenv('FLYWAY_USER')
+  password = System.getenv('FLYWAY_PASSWORD')
+  baselineOnMigrate = true
+  baselineVersion = '000'
+}
+
+task migratePostgresDatabase(type: org.flywaydb.gradle.task.FlywayMigrateTask) {
+  baselineOnMigrate = true
+  if (project.hasProperty("dburl")) {
+    url = "jdbc:postgresql://${dburl}"
+  }
+}
+```
+
+Ensure the RUN_DB_MIGRATION_ON_STARTUP is configured appropriately:
+
+- Add `RUN_DB_MIGRATION_ON_STARTUP: true` to the `values.preview.template.yaml` file
```

Comment:
```suggestion
- Add `RUN_DB_MIGRATION_ON_STARTUP: true` to the `values.preview.template.yaml` file - This is so that preview will migrate on startup
```